### DISCUSSION TITLE
Internet Search

### DISCUSSION THREAD

**Discussion by damizkrli** [dailydotdev/daily#1666] [Feature Requests]
I install Daily.dev and I want to make a search from the homepage. But when I make ctrl+k it's juste possible to search post in the extension. Is it possible to make a feature for choose where we want to search ? i.e. on google or another one ? 
Thanks for your answer

**Comment by nimrodkra** [comment]
This feature already exists. See screenshot (you can search with Google as well)
![Screenshot 2024-12-18 at 23 00 07](https://github.com/user-attachments/assets/0b4bb734-db68-4527-a9f1-49ad03a7ed3c)
